web/templates/layout/app.html.eex

```diff
@@ -12,6 +12,9 @@
   </head>
 
   <body class="<%= if @view_module.include_backdrop, do: "photo-backdrop" %>">
+    <%= if @view_module.include_backdrop do %>
```

Comment:
hey @tnewell5, there's a specific template for the landing page. This is the app.layout.template that _wraps_ all other templates. Put the ribbon in the non-layout template so we can avoid the conditional!